### ISSUE TITLE
Excuse me, could you make an sln version? I am not familiar with cmake.

### ISSUE THREAD

**Issue by Night9zz** [Chaoses-Ib/IbInputSimulator#35]
Sry,as the title says.

**Comment by Chaoses-Ib** [comment]
VS supports CMake natively. Or you can run CMake and use the generated .sln.

**Comment by Night9zz** [comment]
Yes, I successfully generated sln with cmake, but I only found the buildable project lbwincpp, not the IbInputSimulator.
![R7TF6M8U @0BEP3VUNM6JSR](https://github.com/user-attachments/assets/0f8f7ae2-0a63-4ede-bac3-30d4d9b48a1d)
